### PR TITLE
Add algotom package

### PR DESCRIPTION
**Pull request by algotom**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by algotom** [comment]
@jezdez : Should be ready to merge. Many thanks.

**Comment by algotom** [comment]
@ocefpaf : Hi, could you help to check and merge please? It's pretty simple recipe. Many thanks.

File: recipes/algotom/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% set name = "algotom" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/algotom/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipes/algotom/meta.yaml:9]: Any reason to not use the PyPI source?

File: recipes/algotom/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  sha256:  9a72ec5458c1604bbe4ae6e08f148cd27c3671b7df4a6cc9b4fdcda373176fca
+build:  
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt
```
Comment (ocefpaf) [inline recipes/algotom/meta.yaml:14]: You should use `pip` here. See the recipe example or, better yet, re-generate this [recipe with grayskull](https://github.com/conda-incubator/grayskull).

File: recipes/algotom/meta.yaml (hunk 1 above)
Comment (algotom) [reply recipes/algotom/meta.yaml:9]: Just because I modified the recipe from the one I used before. I can change to pypi

**Comment by algotom** [comment]
@ocefpaf: Thanks for the suggestion. I've updated the recipe and things look ok.

**Pull request merged**